src/test/java/apoc/load/relative/LoadXmlRelativePathTest.java

```diff
@@ -0,0 +1,268 @@
+                    childrenList = (List<Object>) children;
+                    assertEquals("bk110", ((Map) result).get("id"));
+                    assertEquals("author", ((Map) childrenList.get(0)).get("_type"));
+                    assertEquals("O'Brien, Tim", ((Map) childrenList.get(0)).get("_text"));
+                    assertEquals("title", ((Map) childrenList.get(1)).get("_type"));
+                    assertEquals("Microsoft .NET: The Programming Bible", ((Map) childrenList.get(1)).get("_text"));
+                    next = r.next();
+                    result = next.get("result");
+                    resultMap = (Map) next.get("result");
+                    children = resultMap.get("_children");
+                    childrenList = (List<Object>) children;
+                    assertEquals("bk111", ((Map) result).get("id"));
+                    assertEquals("author", ((Map) childrenList.get(0)).get("_type"));
+                    assertEquals("O'Brien, Tim", ((Map) childrenList.get(0)).get("_text"));
+                    assertEquals("title", ((Map) childrenList.get(1)).get("_type"));
+                    assertEquals("MSXML3: A Comprehensive Guide", ((Map) childrenList.get(1)).get("_text"));
+                    next = r.next();
+                    result = next.get("result");
+                    resultMap = (Map) next.get("result");
+                    children = resultMap.get("_children");
+                    childrenList = (List<Object>) children;
+                    assertEquals("bk112", ((Map) result).get("id"));
+                    assertEquals("author", ((Map) childrenList.get(0)).get("_type"));
+                    assertEquals("Galos, Mike", ((Map) childrenList.get(0)).get("_text"));
+                    assertEquals("title", ((Map) childrenList.get(1)).get("_type"));
+                    assertEquals("Visual Studio 7: A Comprehensive Guide", ((Map) childrenList.get(1)).get("_text"));
+                    assertEquals(false, r.hasNext());
+                });
+    }
+
+    @Test
+    public void testLoadXmlNoFailOnError () {
+        testCall(db, "CALL apoc.load.xml('books.xm', '', {failOnError:false}) yield value as result",
+                (r) -> {
+                    Map resultMap = (Map) r.get("result");
+                    assertEquals(Collections.emptyMap(), resultMap);
+                });
+    }
+
+    @Test
+    public void testLoadXmlWithImport() {
+        testCall(db, "call apoc.xml.import('humboldt_soemmering01_1791.TEI-P5.xml', {createNextWordRelationships: true}) yield node",
+                row -> {
+                   assertNotNull(row.get("node"));
+                });
+        testResult(db, "match (n) return labels(n)[0] as label, count(*) as count", result -> {
+            final Map<String, Long> resultMap = result.stream().collect(Collectors.toMap(o -> (String)o.get("label"), o -> (Long)o.get("count")));
+            assertEquals(2l, (long)resultMap.get("XmlProcessingInstruction"));
```

Comment:
```suggestion
            assertEquals(2l, resultMap.get("XmlProcessingInstruction").longValue());
```
e a cascata tutti gli altri